app/components/support_interface/personal_details_component.rb

```diff
@@ -40,8 +40,8 @@ def name_row
       {
         key: 'Full name',
         value: "#{first_name} #{last_name}",
-        action: 'Change',
-        action_path: support_interface_application_form_edit_applicant_details_path(application_form),
+        action: 'Change name',
```

Comment:
Close! ‘Change’ is included in the component, so all you need to provide is a description of what can be changed.

```suggestion
        action: 'name',
```